packages/widgets/src/accordionpanel.ts

```diff
@@ -66,6 +66,32 @@ export class AccordionPanel extends SplitPanel {
     widget.title.changed.connect(this._onTitleChanged, this);
   }
 
+  /**
+   * Collapse the widget at position `index`.
+   *
+   * @param index Widget index
+   */
+  collapse(index: number): void {
+    const widget = (this.layout as AccordionLayout).widgets[index];
```

Comment:
We should handle when `widget` is `undefined` because `index` is wrong, I think.